webpack.config.js

```diff
@@ -0,0 +1,61 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+const styleLintPlugin = require('stylelint-webpack-plugin');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
+const path = require('path')
+
+module.exports = {
+  entry: './src/main.js',
+  devtool: 'inline-source-map',
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist')
+  },
+  module: {
+    rules: [
+      {
+        enforce: 'pre',
+        test: /\.jsx?$/,
+        use: ['standard-loader'],
+        exclude: /(node_modules|bower_components)/
+      },
+      {
+        test: /\.pug/,
+        use: ['pug-loader']
+      },
+      {
+        test: /\.scss$/,
+          use: ExtractTextPlugin.extract({
+            use: [
+              {
+                loader: 'css-loader?sourceMap',
```

Comment:
Since you are using the `options` syntax for passing parameters, you can remove the `?sourceMap` flag.